src/authentication/authentication.js

```diff
@@ -0,0 +1,43 @@
+const { sign, verify } = require('jsonwebtoken');
+require('env2')('config.env');
+
+const createCookie = (user, cb) => {
+  sign(user, process.env.SECRET, (signerror, token) => {
+    if (signerror) { cb(signerror); } else {
```

Comment:
Fix your indentation.

Reply:
I cannot see any notifications by eslint regarding indentation at that file.